core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -652,6 +845,47 @@ export class CreateServiceComponent implements OnInit {
     return false;
   };
 
+  hasClass(el, cls) {
+    if (el.className.match('(?:^|\\s)'+cls+'(?!\\S)')) { return true; } 
+    }
+  addClass(el, cls) {
+    if (!el.className.match('(?:^|\\s)'+cls+'(?!\\S)')){ el.className += ' '+cls; } 
+    }
+  delClass(el, cls) {
+    el.className = el.className.replace(new RegExp('(?:^|\\s)'+cls+'(?!\\S)'),'');
+    }
+
+  elementFromTop(elem, classToAdd, distanceFromTop, unit) {
+    var winY = window.innerHeight || document.documentElement.clientHeight,
+        distTop = elem.getBoundingClientRect().top,
+        distPercent = Math.round((distTop / winY) * 100),
+        distPixels = Math.round(distTop),
+        distUnit;
+    distUnit = unit == 'percent' ? distPercent : distPixels;
+    if (distUnit <= distanceFromTop) {
+      if (!this.hasClass(elem, classToAdd)) { this.addClass(elem, classToAdd); }
+      } else {
+      this.delClass(elem, classToAdd);
+      }
+    }
+
+  // newFunc(){
```

Comment:
Can you explain why these comments are included?

Reply:
Removed the comments